data/database.js

```diff
@@ -5,4 +5,8 @@ db.version(1).stores({
   cartItems: `name, price, description, image, count`,
 });
 
+db.version(2).stores({
```

Comment:
why create a different db for this, you could just add `favoriteItems` store in the previous definition along with cartItems, right?

Reply:
I thought i need another db for this favorite item list . So i used this one.